File: WEEK 1/C#/Sum_of_Five_Decimal_Numbers.cs
using System;

class Program
{
    static void Main()
    {
        double sum = 0.0;

        Console.WriteLine("Enter five decimal numbers:");

        for (int i = 0; i < 5; i++)
        {
            double num;
            if (double.TryParse(Console.ReadLine(), out num))
            {
                sum += num;
            }
            else
            {
                Console.WriteLine("Invalid input. Please enter a valid decimal number.");
                i--; // Decrement i to re-enter the current value
            }
        }

        int roundedSum = (int)(sum + 0.5);
        Console.WriteLine($"Sum rounded to the nearest integer: {roundedSum}");
    }
}

File: WEEK 1/java/Sum_of_Five_Decimal_Numbers.java
import java.util.Scanner;

public class DecimalSumToNearestInteger {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        
        System.out.println("Enter five decimal numbers:");
        
        double sum = 0.0;
        
        for (int i = 0; i < 5; i++) {
            System.out.print("Number " + (i + 1) + ": ");
            double number = input.nextDouble();
            sum += number;
        }
        
        input.close();
        
        // Round the sum to the nearest integer
        int roundedSum = (int) Math.round(sum);
        
        System.out.println("Sum of the numbers: " + sum);
        System.out.println("Sum rounded to the nearest integer: " + roundedSum);
    }
}